<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <layout class="QHBoxLayout" name="mainLayout">
   <item>
    <layout class="QVBoxLayout" name="leftColumn">
     <item>
      <widget class="QPushButton" name="openFileButton">
       <property name="text">
        <string>Ouvrir M3U</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchBar">
       <property name="placeholderText">
        <string>Recherche...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="filterCountry"/>
     </item>
     <item>
      <widget class="QComboBox" name="filterCategory"/>
     </item>
     <item>
      <widget class="QListWidget" name="channelsList"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="centerColumn">
     <item>
      <widget class="QLabel" name="logoLabel">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="videoFrame">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="minimumSize">
        <size>
         <width>640</width>
         <height>360</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="controlsLayout">
       <item>
        <widget class="QPushButton" name="playButton">
         <property name="text">
          <string>‚ñ∂</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pauseButton">
         <property name="text">
          <string>‚è∏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="text">
          <string>‚èπ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="volumeSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="muteButton">
         <property name="text">
          <string>üîá</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="progressSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="rightColumn">
     <item>
      <widget class="QTextEdit" name="epgWidget">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>EPG ici...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="favoritesList">
       <property name="toolTip">
        <string>Vos cha√Ænes favorites</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
